### PR TITLE
Add tutorials for a first utilisation of FrontLine, closes MISC-52

### PR DESCRIPTION
**Pull request by ccedric**
Motivation:
- users should have an easy documentation in one page on how to begin

Modifications:
- created the tutorials under /tutorials, available from the navbar
- added tutorials for Maven, gradle, sbt and bundle

### CODE REVIEW

File: archetypes/blog.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ date: {{ .Date }}
 lastmod: {{ .Date }}
 draft: true
 weight: 50
-images: ["{{ .Name | urlize }}.jpg"]
+images: []
```
Comment (Isammoc) [inline archetypes/blog.md:9]: why?

File: content/blog/_index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: "Blog"
```
Comment (Isammoc) [inline content/blog/_index.md:2]: why?

```suggestion
title: "Tutorials"
```

File: config/_default/menus.toml [hunk 3]
```diff
@@ -16,6 +16,12 @@
   identifier = "plugins"
   url = "/docs/plugins/"
 
+[[main]]
+  name = "Tutorials"
+  weight = 20
+  identifier = "tutorials"
+  url = "/blog/"
```
Comment (Isammoc) [inline config/_default/menus.toml:23]: Cannot we use them as?

```suggestion
  url = "/tutorials/"
```

File: content/blog/gradle-tutorial/index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+---
+title: "FrontLine with Gradle Tutorial"
+description: "step by step setup of FrontLine"
+lead: ""
```
Comment (Isammoc) [inline content/blog/gradle-tutorial/index.md:4]: remove empty values

File: content/blog/gradle-tutorial/index.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+date: 2021-03-19T17:03:31+01:00
+lastmod: 2021-03-19T17:03:31+01:00
+draft: false
+images: []
+weight: 50
+images: []
```
Comment (Isammoc) [inline content/blog/gradle-tutorial/index.md:10]: duplicate

File: content/blog/maven-tutorial/index.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
+---
+title: "FrontLine with Maven Tutorial"
+description: "step by step setup of FrontLine"
+lead: ""
+date: 2021-03-19T17:03:31+01:00
+lastmod: 2021-03-19T17:03:31+01:00
+draft: false
+images: []
+weight: 50
+images: []
```
Comment (Isammoc) [inline content/blog/maven-tutorial/index.md:10]: duplicate

File: content/blog/sbt-tutorial/index.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+---
+title: "FrontLine with Sbt Tutorial"
+description: "step by step setup of FrontLine"
+lead: ""
+date: 2021-03-19T17:03:31+01:00
+lastmod: 2021-03-19T17:03:31+01:00
+draft: false
+images: []
+weight: 50
+images: []
```
Comment (Isammoc) [inline content/blog/sbt-tutorial/index.md:10]: duplicate

File: content/tutorials/simulation.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+## Additional resources
+
+You have successfully started load testing, you can read our documentation if you need more information about our product:
+
+FrontLine documentation: https://beta.gatling.io
+
+Documentation about the creation of an artifact: https://beta.gatling.io/docs/user/artifact_gen
+
+Documentation about how to use the FrontLine reports: https://beta.gatling.io/docs/user/reports/
```
Comment (Isammoc) [inline content/tutorials/simulation.md:51]: Links are not generated (perhaps because of the inclusion?)

File: content/blog/_index.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+lead: ""
+date: 2021-03-19T17:03:04+01:00
+lastmod: 2021-03-19T17:03:04+01:00
+draft: false
+weight: 50
+images: []
+contributors: []
+---
+
+Blog
```
Comment (Isammoc) [inline content/blog/_index.md:13]:
```suggestion
Tutorials
```

File: README.md [hunk 10]
```diff
@@ -13,3 +13,11 @@ npm run server
 ```
```
Comment (Isammoc) [inline README.md:13]: bundle tutorial?

**Review by Isammoc — COMMENTED** [review]
:clap: 

Huge work!

:tada:

File: content/blog/_index.md (hunk 2 above)
Comment (ccedric) [reply content/blog/_index.md:2]: The tutorials are atm in the "blog" part, and we only have the tutorials, so I did this change.
We need to decide if we leave the tutorials here, and if we're gonna add some other things to blog.

File: content/blog/gradle-tutorial/index.md (hunk 5 above)
Comment (ccedric) [reply content/blog/gradle-tutorial/index.md:10]: I need to add this conf in each file

File: README.md (hunk 10 above)
Comment (ccedric) [reply README.md:13]: He is in progress

File: content/blog/gradle-tutorial/index.md (hunk 5 above)
Comment (Isammoc) [reply content/blog/gradle-tutorial/index.md:10]: `images` entry is duplicate in the same file

File: content/tutorials/simulation.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+<img src="images/reports.png" alt="FrontLine Reports" width="900"/>
+
+You will find here every information you may need to analyze your run. Take some time to interact with the graphs and change tabs.
+
+## Additional resources
+
+You have successfully started load testing, you can read our documentation if you need more information about our product:
+
+FrontLine documentation: https://beta.gatling.io
```
Comment (notdryft) [inline content/tutorials/simulation.md:47]:
```suggestion
FrontLine documentation: {{< ref "/" >}}
```

Comment (notdryft) [reply content/tutorials/simulation.md:47]: See https://gohugo.io/content-management/cross-references/

File: content/tutorials/simulation.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+Documentation about the creation of an artifact: https://beta.gatling.io/docs/user/artifact_gen
```
Comment (notdryft) [inline content/tutorials/simulation.md:49]:
```suggestion
Documentation about the creation of an artifact: {{< ref "docs/user/artifact_gen" >}}
```

Comment (notdryft) [reply content/tutorials/simulation.md:49]: This reminds me `artifact_gen` is underscore whereas all the pages here are dashed. Should we dash all the pages?

File: content/tutorials/simulation.md (hunk 8 above)
Comment (notdryft) [inline content/tutorials/simulation.md:51]:
```suggestion
Documentation about how to use the FrontLine reports: {{< ref "docs/user/reports/" >}}
```

**Review by notdryft — CHANGES_REQUESTED** [review]

File: content/tutorials/simulation.md (hunk 12 above)
Comment (ccedric) [reply content/tutorials/simulation.md]: i think dash is better

File: content/blog/_index.md (hunk 2 above)
Comment (ccedric) [reply content/blog/_index.md]: we renamed the section to tutorials

File: content/tutorials/bundle/index.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+
+## Create an artifact
+
+There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run. We can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+
+In this tutorial, we'll use the Gatling bundle. Make sure that you have a JDK 8 or 11 installed on your computer.
+
+Download the latest version of the Gatling bundle here: https://gatling.io/open-source/start-testing and extract the archive.
+
+If you are on a Windows computer, copy this [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) file in the bin directory of the decompressed archive. If you are on a Linux or MacOs computer, copy this [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) file.
```
Comment (Isammoc) [inline content/tutorials/bundle/index.md:24]:
```suggestion
If you are on a Windows computer, copy this [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) file in the `bin` directory of the decompressed archive. If you are on a Linux or MacOs computer, copy this [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) file in the `bin` directory of the decompressed archive.
```
```suggestion
On a Windows computer: copy this [`artifact.bat`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.bat) file in the `bin` directory of the decompressed archive.

On a Linux or MacOS computer: copy this [`artifact.sh`](https://raw.githubusercontent.com/gatling/gatling/master/gatling-bundle/src/universal/bin/artifact.sh) file in the `bin` directory of the decompressed archive.
```

File: content/tutorials/bundle/index.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+{{< img src="bundle.png" alt="Launch artifact.bat" >}}
+
+Run the `artifact.bat` or `artifact.sh` file, Gatling will start creating our first artifact !
```
Comment (Isammoc) [inline content/tutorials/bundle/index.md:28]:
```suggestion
Run the `artifact.bat` or `artifact.sh` file, Gatling will start creating our first artifact!
```

File: content/tutorials/bundle/index.md [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+
+{{< include upload.md "target/artifact.jar" >}}
+
+Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
```
Comment (Isammoc) [inline content/tutorials/bundle/index.md:32]: No text to open the wanted modal?

File: content/tutorials/bundle/index.md [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+{{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}
+
+We now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use !
```
Comment (Isammoc) [inline content/tutorials/bundle/index.md:36]:
```suggestion
We now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use!
```

File: content/tutorials/maven/index.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+
+Execute the following command to create an artifact:
+
+```shell
+mvn clean package -DskipTests
+```
+
+{{< img src="maven_command.png" alt="Maven command" >}}
+
+Maven will download the necessary dependencies and package our simulation. That's it, we created our first Gatling artifact !
```
Comment (Isammoc) [inline content/tutorials/maven/index.md:33]:
```suggestion
Maven will download the necessary dependencies and package our simulation. That's it, we created our first Gatling artifact!
```

File: content/tutorials/maven/index.md [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+
+{{< include upload.md "frontline-samples/maven/target/maven-sample-1.0.0-shaded.jar" >}}
+
+Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+
+{{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}
+
+We now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use !
```
Comment (Isammoc) [inline content/tutorials/maven/index.md:41]:
```suggestion
We now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use!
```

File: content/tutorials/sbt/index.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+
+Execute the following command to create an artifact:
+
+```shell
+sbt test:assembly
+```
+
+{{< img src="sbt_command.png" alt="sbt command" >}}
+
+sbt will download the necessary dependencies and package our simulation. That's it, we created our first Gatling artifact !
```
Comment (Isammoc) [inline content/tutorials/sbt/index.md:33]:
```suggestion
sbt will download the necessary dependencies and package our simulation. That's it, we created our first Gatling artifact!
```

File: content/tutorials/sbt/index.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+
+{{< include upload.md "frontline-samples/sbt/target/sbt-frontline-1.0.0.jar" >}}
+
+Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+
+{{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}
+
+We now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use !
```
Comment (Isammoc) [inline content/tutorials/sbt/index.md:41]:
```suggestion
We now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use!
```

**Review by Isammoc — APPROVED** [review]
Some nits, but LGTM

File: content/tutorials/bundle/index.md (hunk 15 above)
Comment (ccedric) [reply content/tutorials/bundle/index.md:32]: The text is in the included upload.md

**Pull request merged**